src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/PersonAddress.java

```diff
@@ -0,0 +1,55 @@
+package uk.nhs.digital.nhsconnect.nhais.model.edifact;
+
+import lombok.Builder;
+import lombok.Data;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.message.EdifactValidationException;
+
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Builder
+@Data
+public class PersonAddress extends Segment {
+    private final static String NAME_AND_ADDRESS = "NAD";
+    private final static String PAT_CODE = "PAT";
+    private final static String COLON_SEPARATOR = ":";
+    private String addressLine1;
+    private String addressLine2;
+    private String addressLine3;
+    private String addressLine4;
+    private String addressLine5;
+
+    @Override
+    public String getKey() {
+        return NAME_AND_ADDRESS;
+    }
+
+    @Override
+    public String getValue() {
+        String address = Stream.of(addressLine1, addressLine2, addressLine3, addressLine4, addressLine5)
+            .filter(Objects::nonNull)
+            .map(String::toUpperCase)
```

Comment:
I think we've agreed that we won't modify incoming data but enforce GP to provide proper EDIFACT compliant data